Training/etc/webapi.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+-->
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Webapi:etc/webapi.xsd">
+    <route url="/V2/customers/:customerId" method="POST">
+        <service class="Codifi\Training\Api\UpdateCustomerAttributeCreditHoldInterface" method="getPost"/>
```

Comment:
It's better to use more abstract class name and more precise and clear method name. For example, class name `CreditHoldAttributeManagement` and method name `updateAttribute`